admin/class-helpers.php

```diff
@@ -211,4 +211,21 @@ function ( $rule ) {
 		}
 		return $results;
 	}
+
+	/**
+	 * Do a capability check for the current user to ensure they have the required capability
+	 * to see various widgets or notices.
+	 *
+	 * @since 1.9.3
+	 */
+	public static function current_user_can_see_widgets_and_notices(): bool {
```

Comment:
Shouldn't we create a unit test for this method?

Reply:
Good idea, I added tests for expected values and also a case that covers what would be expected should the value be filtered to a given cap